src/app.py

```diff
@@ -91,6 +76,41 @@ def connect(self, **kwargs):
         except Exception as e:
             print(f"Error: {e}")
 
+    # Note: To connect with Couchbase Capella, use the following connect() instead of the one above as it communicates with TLS.
+    # This example does not use certificates for authentication. In production, you should have it enabled.
+    # Also ensure that the bucket, scopes & collections are created on Capella before running the application.
```

Comment:
Nope, only the bucket should be present while using capela or a local server. In both cases we must automatically create the required indexes, scopes and collections.